wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SwerveControllerCommand.java

```diff
@@ -69,42 +63,91 @@
    *                           for the robot's y position.
    * @param thetaController    The Trajectory Tracker PID controller
    *                           for angle for the robot.
+   * @param desiredRotation    The angle that the drivetrain should be facing. This
+   *                           is sampled at each time step.
    * @param outputModuleStates The raw output module states from the
    *                           position controllers.
    * @param requirements       The subsystems to require.
    */
-
   @SuppressWarnings("ParameterName")
   public SwerveControllerCommand(Trajectory trajectory,
-                               Supplier<Pose2d> pose,
-                               SwerveDriveKinematics kinematics,
-                               PIDController xController,
-                               PIDController yController,
-                               ProfiledPIDController thetaController,
-
-                               Consumer<SwerveModuleState[]> outputModuleStates,
-                               Subsystem... requirements) {
+                                 Supplier<Pose2d> pose,
+                                 SwerveDriveKinematics kinematics,
+                                 PIDController xController,
+                                 PIDController yController,
+                                 ProfiledPIDController thetaController,
+
+                                 Supplier<Rotation2d> desiredRotation,
+
```

Comment:
Are these blank lines here on purpose?

Reply:
Not really; that's how it was before, but I can remove them since they don't really help much.